alerts/report_sources/goma/alerts_goma_2019-10-26.Rmd

```diff
@@ -0,0 +1,2139 @@
+---
+title: "Investigation of alerts data: Goma"
+author: "Charlie Whittaker, Thibaut Jombart, Flavio Finger, Christopher Jarvis, Jonathan Polonsky, Patrick Keating, and Amy Gimma for the analytic cell OEC Goma"
+date: "`r format(Sys.time(), '%A %d %B %Y')`"
+output:
+  html_document: 
+    code_folding: hide
+    highlight: zenburn
+    number_sections: yes
+    theme: spacelab
+    toc: yes
+    toc_collapse: no
+    toc_depth: 1
+    toc_float: yes
+    css: !expr here::here('css', 'style.css')
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(echo = TRUE,
+                      collapse = TRUE,
+                      fig.width = 8,
+                      fig.height = 6, 
+                      dpi = 150,
+                      warning = FALSE,
+                      message = FALSE,
+                      fig.path = "figures/")
+```
+
+<!-- ======================================================= -->
+<!-- ======================================================= -->
+<!-- ======================================================= -->
+
+# Data preparation {.tabset .tabset-fade .tabset-pills}
+
+## Outline
+
+This report cleans and analyses the alert data of Goma. Input comes from an
+`xlsx` file containing alerts in a specific format. Because all
+sub-coordinations have different standards, each sub-coordination needs a
+separate report.
+
+The data preparation involves the following steps, detailed in the following 
+tabs:
+
+* **Load scripts**: loads libraries and useful scripts used in the analyses; all
+  `.R` files contained in `scripts` at the root of the factory are automatically
+  loaded
+
+* **Load data**: imports datasets, and may contain some *ad hoc* changes to the
+data such as specific data cleaning (not used in other reports), new variables
+used in the analyses, etc.
+
+* **Clean data**: this section contains *ad hoc* data cleaning, i.e. which is
+  not used in other reports (otherwise cleaning should be done in a dedicated
+  report); this section is also used to create new variables used in the
+  analyses
+
+## Load scripts
+
+These scripts will load:
+
+* all local scripts, stored as `.R` filesinside `/scripts/`
+* all Overall scripts, i.e. stored outside the factory in `../scripts/`
+
+```{r read_scripts}
+
+## read scripts
+path_to_scripts <- here::here("scripts")
+scripts_files <- dir(path_to_scripts, pattern = ".R$",
+                     full.names = TRUE)
+for (file in scripts_files) source(file)
```

Comment:
Use `souce(file, local = TRUE)` to make sure `params` are passed across to scripts.